### PR TITLE
Chaincode support

### PR DESCRIPTION
**Pull request by mvanmeerbeck**

### CODE REVIEW

File: src/orchestrator.ts [hunk 1]
```diff
@@ -63,6 +69,9 @@ import getArtifactsPath = Utils.getArtifactsPath;
  */
 export class Orchestrator {
 
+    public cliGenerator;
+  //  public targetPeers;
```
Comment (mvanmeerbeck) [inline src/orchestrator.ts]: remove commented code

File: src/generators/docker-compose/dockercomposepeer.yaml.ts [hunk 2]
```diff
@@ -78,7 +78,7 @@ ${this.options.org.peers
       - ${this.options.networkRootPath}/organizations/peerOrganizations/${this.options.org.fullName}/peers/${peer.name}.${this.options.org.fullName}/msp:/etc/hyperledger/fabric/msp
       - ${this.options.networkRootPath}/organizations/peerOrganizations/${this.options.org.fullName}/peers/${peer.name}.${this.options.org.fullName}/tls:/etc/hyperledger/fabric/tls
       - ${peer.name}.${this.options.org.fullName}:/var/hyperledger/production
-${this.options.ips.length > 0 ? `
+${this.options.ips ? `
```
Comment (mvanmeerbeck) [inline src/generators/docker-compose/dockercomposepeer.yaml.ts]: does it returns true if the array is empty ? 
ips: []

File: src/command.ts [hunk 3]
```diff
@@ -239,6 +251,62 @@ channelCmd
     .action(async (cmd) => {
       await tasks.updateChannel(cmd.anchortx, cmd.namech, cmd.config);
     });
+
+//peers, chaincodeName, chaincodePath, chaincodeVersion
+//sudo bnc chaincode install -n mycc -v v0 -p p1,p2  -f ./tests/manual/wassim/config-deploy-org2.yaml
```
Comment (mvanmeerbeck) [inline src/command.ts]: remove commented code

File: src/generators/docker-compose/dockercomposepeer.yaml.ts (hunk 2 above)
Comment (sahar-fehri) [reply src/generators/docker-compose/dockercomposepeer.yaml.ts]: if the ips section does not exist it wont proceed to creating extra-hosts section

File: src/orchestrator.ts (hunk 1 above)
Comment (sahar-fehri) [reply src/orchestrator.ts]: done

File: src/generators/docker-compose/dockercomposepeer.yaml.ts (hunk 2 above)
Comment (sahar-fehri) [reply src/generators/docker-compose/dockercomposepeer.yaml.ts]: fixed by : ${this.options.ips && this.options.ips.length > 0 ?  `

**Pull request merged**